etc/nvliblist.conf

```diff
@@ -1,9 +1,12 @@
 # NVLIBLIST.CONF
 # This configuration file determines which NVIDIA libraries to search for on 
 # the host system when the --nv option is invoked.  You can edit it if you have
-# different libraries on your host system.
+# different libraries on your host system.  You can also add binaries and they
+# will be mounted into the container when the --nv option is passed.
 
 # put binaries here
+# In shared environments you should ensure that permissions on these files 
+# exclude writing by non-privileged users.  
```

Comment:
+1!

Reply:
>> Yeah. I guess. Then we will have to start every bash script with a bunch of include like statements similar to some other languages I know...

But not python, because python uses "from" and "import" and because python is just flawless! :P